index.html

```diff
@@ -139,55 +184,61 @@ <h2 class="title">Multi-Post Stories</h2>
     </section>
     <!-- end of work section-->
 
+    <!-- About section for mobile -->
     <section class="about">
-      <h2 class="headline_title">About Myself</h2>
-      <p class="about_details">
-        Hello I’m a software developer! I can help you build a product ,
-        feature or website Look through some of my work and experience! If you
-        like what you see and have a project you need coded, don’t hestiate to
-        contact me.
-      </p>
-      
-      <div class="btn-link"><a href="#"><h3 class="connect_button">let's connect</h3></a></div>
+      <div class="about-div">
+          <h2 class="headline_title">About Myself</h2>
+          <p class="about_details">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
 
-        <div class="social">
-          <ul>
-            <li><a href="#"><img src="./assets/Vector.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a></li>
-            <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
-            <li><a href="#"><img src="./assets/Vector-1.svg" alt="github"></a></li>
-            <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
-          </ul>
-        </div>
-      <div class="action">
-        <button type="button" class="default_button">Get my resume</button>
-      </div>
-      <ul class="about_ul">
+          <div class="btn-link" id="about-connect-link"><a href="#"><h3 class="connect_button">let's connect</h3></a></div>
+
+              <div class="social" id="about-social-links">
+                <ul>
+                  <li><a href="#"><img src="./assets/Vector.svg" alt="twitter"></a></li>
+                  <li><a href="#"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a></li>
+                  <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
+                  <li><a href="#"><img src="./assets/Vector-1.svg" alt="github"></a></li>
+                  <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
+                </ul>
+              </div>
+          
+              <div class="action">
+                <button type="button" class="default_button">Get my resume</button>
+              </div>
+          </div>
+    </div>
+      <div class="languages">
```

Comment:
Good job! but as you know it's required to follow the original Figma design, so please make sure to consider the below points.
1. The top-right corner of the about section is rounded
2. You have some extra space to the right in the **Languages** section collapse. Please remove it.

![image](https://user-images.githubusercontent.com/34779928/200076348-d59b76ab-0e06-4866-87c0-286bc0bb50b1.png)